README.md

```diff
@@ -220,6 +253,46 @@ typical setup.
 }
 ```
 
+<a name="performance"></a>
+## Performance
+
+<a name="specificity"></a>
+### Specificity
+
+Although in the name (cascading style sheets) cascading can introduce
+unnecessary performance overhead for applying styles. Take the following
+example:
+
+```css
+ul.user-list li span a:hover { color: red; }
+```
+
+Styles are resolved during the renderer's layout pass. The selectors are
+resolved right to left, exiting when it has been detected the selector does not
+match. Therefore, in this example every a tag has to be inspected to see if it
+resides inside a span and a list. As you can imagine this requires a lot of DOM
+walking and and for large documents can cause a significant increase in the
+layout time. For further reading checkout:
+https://developers.google.com/speed/docs/best-practices/rendering#UseEfficientCSSSelectors
+
+If we know we want to give all `a` elements inside the `.user-list` red on hover
+we can simplify this style to:
+
+```css
+.user-list > a:hover {
+    color: red;
+}
+```
```

Comment:
Well, I don't see it as a separate issue because the existence of the example suggests that css with  knowledge of the html is okay, which, correct me if I'm wrong, that's not something we want to do, as the objective of BEM is to be largely dom agnostic. 
Also, how is mine less performant, mine is just a class selector, where as yours has inheritance to calculate.